lib/jekyll/pagewithoutafile.rb

```diff
@@ -0,0 +1,7 @@
+module Jekyll
```

Comment:
Regardless of the above, this should remain in `lib/jekyll`, as it's an extension of Jekyll, not jekyll-feed.